mod_reforged/hooks/config/tip_of_the_day.nut

```diff
@@ -19,13 +19,13 @@ for (local index = (::Const.TipOfTheDay.len() - 1); index >= 0; index--)
 }
 
 ::Const.TipOfTheDay.extend([
-	"You can enter towns while on a caravan mission, even if they are 2 tiles away.",
+	"Reforged: You can enter towns while on a caravan mission, even if they are 2 tiles away.",
```

Comment:
The first part of this is a vanilla mechanic. Only the second part is new.
What should happen in this case?

Reply:
I find that an improvement